### PR TITLE
Report perftests performance differences via emails

### PR DESCRIPTION
**Pull request by ChGen**
Report perftests performance differences via emails. Part of https://github.com/artipie/benchmarks/issues/64

### CODE REVIEW

**Comment by ChGen** [comment]
@dgarus , please review

File: .github/workflows/perftest-client.yml [hunk 1]
```diff
@@ -15,11 +15,11 @@ jobs:
           ref: ${{ github.event.workflow_run.head_branch }}
       - run: env;git branch -a; git status
       - name: Check connection
-        run: . "/opt/.benchvars" && curl -v "http://$SERVER_HOST:$REPO_PORT/bintest"
+        run: . "/opt/.benchvars" && timeout 30 curl -v "http://$SERVER_HOST:$REPO_PORT/bintest"
       - name: Check variables
         run: test ! -f /opt/.benchvars && exit 1;exit 0
       - name: Checkout benchmarks repo
-        run: git clone --depth=1 https://github.com/artipie/benchmarks.git
```
Comment (dgarus) [inline .github/workflows/perftest-client.yml:22]: Why do we change the repo?

Comment (ChGen) [reply .github/workflows/perftest-client.yml]: @dgarus , thanks, I forgot to revert it back after testing.

**Review by olenagerasimova — DISMISSED** [review]

**Comment by ChGen** [comment]
@dgarus , please review, I removed redundant TODOs and rerun checks.

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**